### DISCUSSION TITLE
Developer Note:  Image widget when prefixing the path with "assets"

### DISCUSSION THREAD

**Discussion by jamestagal** [plentico/plenti#254] [General]
Hi all,

I just wanted to bring to your attention a note about the image widget in the Plenti CMS. By design, the CMS 'discovers' your images when the source is prefixed with the path ```assets``` in your JSON data source not in the HTML template as follows:
JSON
```json
  "image": {
        "source": "assets/post-1.jpeg",
    },
```
HTML
```html
<img src="{image.source}" alt="{image.alt}" class="img-fluid w-100">
```
RESULT
<img width="1378" alt="Screen Shot 2023-01-09 at 12 13 01 pm" src="https://user-images.githubusercontent.com/6309595/211228698-8dba6a8f-21da-48e9-a0e0-d9527d07c498.png">

If on the other hand you wanted to describe an image path in text and not have the image widget, you would do the opposite.
JSON
```json
  "image": {
        "source": "post-1.jpeg",
    },
```
HTML
```html
<img src="assets/{image.source}" alt="{image.alt}" class="img-fluid w-100">
```
RESULT
<img width="1378" alt="Screen Shot 2023-01-09 at 12 16 54 pm" src="https://user-images.githubusercontent.com/6309595/211229001-2afb69fd-3b0f-412d-b044-fc35de3f8f99.png">

See a discussion on this topic. [A case when image picker not being handled in CMS #253](https://github.com/plentico/plenti/issues/253)

Ben

**Comment by jimafisk** [comment]
This is great @jamestagal, thanks for adding the guide! One quick note, the `v0.6.x` release is coming out very soon and we're changing `/assets/` to `/media/`.
